Formula/gmt.rb

```diff
@@ -1,11 +1,11 @@
 class Gmt < Formula
   desc "Tools for processing and displaying xy and xyz datasets"
   homepage "https://gmt.soest.hawaii.edu/"
```

Comment:
```suggestion
  homepage "https://www.generic-mapping-tools.org"
```
We're switching to the new domain. Currently, it's redirected to http://docs.generic-mapping-tools.org